devops/dev/config/application-dev.properties

```diff
@@ -17,7 +17,7 @@ banner.job.interval=0 0/5 * * * ?
 saml.idp.metadata.location=file:${application.dir}/config/idp-eleg.xml
 
 server.port=8041
-
+environment=staging
```

Comment:
As discussed:
1) Alphabetic order
2) environment -> application.environment
3) default "prod" in application.properties